package.json

```diff
@@ -56,5 +56,9 @@
     "typescript": "^5.5.4"
   },
   "prettier": "@spotify/prettier-config",
+  "volta": {
```

Comment:
I cannot merge this, but I think it would be a lot easier to merge if it was only doing what it said in the title. Perhaps add this tool/feature/functionality in another PR? Just sharing my thoughts.

Reply:
I actually ran a lot of problems because of mismatched versions of different tools like yarn, node and python.

The idea is to elevate this to some degree. If that is the only problem preventing a merge, I happily remove it.

Alternatively there should be something like `.tools-version` which documents the actually used versions so tooling can pick it up.

Reply:
Thanks for the explanation secustor. At this point it's unlikely that we are tying ourselves into a random tool manager.